examples/webpack2-react-simple/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf8" />
+    </head>
+    <body>
+        <div id="wrapper" />
```

Comment:
There's no such thing as self-closing tags in HTML, you should do `<div></div>` to avoid malformed markup.